.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -23,11 +24,14 @@ jobs:
     with:
       default_runner: ubuntu-latest
 
+  check_not_draft_pr:
+    uses: ./.github/workflows/check-for-draft-pr.yaml
+
```

Comment:
And in this action putting it before `determine_runner` and `check_if_wheel_build`.